feat(rust): update rust api examples with colored output

nazmulidris opened the pull request:
<!-- Thank you for sending a pull request :heart: -->

Update Rust API examples so that they have nice colorized output that is easy for developers to read and understand.

## Checks

<!-- To help us review and merge this pull request quickly, please confirm the following:  -->

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

<!-- Looking forward to merging your contribution!! -->

Commit:
feat(rust): update rust api examples with colored output

mrinalwadhwa commented on examples/rust/get_started/examples/01-node.rs:
```diff
@@ -1,12 +1,35 @@
+
+    print_title(vec!["Run a node & stop it right away"]);
+
+    // Create a node with default implementations.
```
```suggestion
    // Create a node.
```

nazmulidris replied:
Done

mrinalwadhwa commented on examples/rust/get_started/examples/01-node.rs:
```diff
@@ -1,12 +1,35 @@
+│  Node 1              │
+├──────────────────────┤
+│  ┌────────────────┐  │
+│  │ Address:       │  │
```
```suggestion
│  │ Worker Address │  │
```

nazmulidris replied:
Done!

mrinalwadhwa reviewed (dismissed):
@nazmulidris this looks great, left a couple of minor suggestions.

nazmulidris commented:
@mrinalwadhwa 
- Thank you for your suggestions 🙏🏽. I've incorporated them all. 
- I've also switched to using a different dependency for ansi 256 color output (that works on linux, macos, and windows & has a compatible license).

> Also, the `lint_docs_ockam_io_rust_examples` are failing in this PR because I've not merged this PR in the `ockam-documentation` repo yet https://github.com/build-trust/ockam-documentation/pull/56

nazmulidris reviewed (commented):
@mrinalwadhwa I've made all the changes you requested 👍🏽
